backend/src/graphql/schema.ts

```diff
@@ -12,5 +12,13 @@ export const schema = async (): Promise<GraphQLSchema> => {
     resolvers: [path.join(__dirname, 'resolver', `!(*.test).{js,ts}`)],
     authChecker: isAuthorized,
     scalarsMap: [{ type: Decimal, scalar: DecimalScalar }],
+    validate: {
+      validationError: { target: false },
+      skipMissingProperties: true,
+      skipNullProperties: true,
+      skipUndefinedProperties: true,
+      forbidUnknownValues: false,
+      stopAtFirstError: false,
```

Comment:
are these the correct configurations for the wanted behaviour?
- do we want to skip missing properties?
- do we want to skip null properties?
- do we want to skip undefined properties?
- do we want to allow unknown values?
- do we want to go on after first error?

Reply:
- do we want to skip missing properties? 
we can change that now that I have added validation checks to all parameters in this version
- do we want to skip null properties?
if they are zero they cannot be validated, can they? I think we have some nullable fields.
- do we want to skip undefined properties? 
we can change that now that I have added validation checks to all parameters in this version
- do we want to allow unknown values?
hmm, you're right, actually we don't need that
- do we want to go on after first error?
we don't need to, but maybe more information could help find the error